go/teams/rpc_exim.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2015 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+// Export-Import for RPC for Teams
+
+package teams
+
+import (
+	keybase1 "github.com/keybase/client/go/protocol/keybase1"
+)
+
+func (t *Team) ExportToTeamPlusAllKeys(idTime keybase1.Time) (*keybase1.TeamPlusAllKeys, error) {
+	perTeamKeys := t.Chain.inner.PerTeamKeys
```

Comment:
Kept as `map<int, PerTeamKey>` to be consistent with the rest of the avdls, but I can change int to Seqno if needed